tools/data-model-exporter/data_model_exporter/__init__.py

```diff
@@ -0,0 +1,33 @@
+__version__ = '0.1.0'
+#!/usr/bin/env python
+
+# Create a basic CLI tool that parses the arg list defined in the spec
```

Comment:
This should be a docstring `"""` vs `#` (see https://www.python.org/dev/peps/pep-0257/if you're curious)